rust-runtime/inlineable/Cargo.toml

```diff
@@ -23,11 +23,11 @@ repository = "https://github.com/awslabs/smithy-rs"
 "pin-project-lite" = "0.2"
 "tower" = { version = "0.4.11", default_features = false }
 "async-trait" = "0.1"
+regex = "1"
```

Comment:
Hmmm... It looks like we should be using a minimum of 1.5.5: https://rustsec.org/advisories/RUSTSEC-2022-0013.html

I guess we must have a dependency that forces upgrade to that version since the minimal version audit hasn't caught this.

Would also need to be fixed here: https://github.com/awslabs/smithy-rs/blob/462aef63945883d9567b2e8cd2be324016f8111e/codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/rustlang/CargoDependency.kt#L205